src/KeyworkerDashboard/KeyworkerDashboard.styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'react-emotion'
+import { SPACING } from '@govuk-react/constants'
+
+// eslint-disable-next-line
```

Comment:
Was moaning about default export as it's the only thing that gets exported currently. More styles will be added so added this line temporarily.